site/nav.yml

```diff
@@ -63,7 +63,6 @@ nav:
         - Apache Amoro: integrations/amoro.md
         - Apache Doris: https://doris.apache.org/docs/dev/lakehouse/catalogs/iceberg-catalog
         - Apache Druid: https://druid.apache.org/docs/latest/development/extensions-contrib/iceberg/
```

Comment:
```suggestion
        - Apache Druid: https://druid.apache.org/docs/latest/development/extensions-contrib/iceberg/
        - BladePipe: https://www.bladepipe.com/
```

Reply:
That address doesn't resolve for me, so I think we probably just want to remove it.